app/src/main/res/layout/fragment_movies_list.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    tools:context=".MoviesListFragment">
+
+
+    <androidx.recyclerview.widget.RecyclerView
```

Comment:
When you have only one view in a layout there is no need to wrap it in a container, here you can directly use the Recycler and remove the RelativeLayout